app/schema.py

```diff
@@ -81,10 +84,10 @@ class MenteeUpdate(BaseModel):
     list_convictions: Optional[constr(max_length=255)]
     tech_stack: Optional[constr(max_length=255)]
```

Comment:
I think this would be better as an arry vs a string if the user can select multiple options.

Reply:
I agree. Can Web send us an array based on the input from the users?

Reply:
Yes! Currently, this input is a series of checkboxes. This makes it so users can select multiple. We can restrict the checkboxes to select a single box or update the html object to radio buttons. With radio buttons, you can only select one. If the stakeholder is certain they want Mentees to only select one "tech_stack", we can make this change on the FE. Additionally, we will need to update the initialFormValue for "tech_stack" to be an array instead of a string.